### DISCUSSION TITLE
"Email address to receive a copy of each invoice" has no effect

### DISCUSSION THREAD

**Discussion by ginkel** [pretix/pretix#4509] [Q&A]
### Problem and impact

We configured our event with the setting "Invoicing" -> "Email address to receive a copy of each invoice" so that copies of all invoices are automatically sent to our document archive. However, I can't find any attempt by Pretix to send such an e-mail. Regular order confirmation mails are dispatched without any issues.

### Expected behaviour

Invoices are sent to the configured e-mail address.

### Steps to reproduce

1. Set up an event with "Invoicing" -> "Email address to receive a copy of each invoice" configured
2. Book a ticket
3. The e-mail is never sent

### Operating system, dependency versions

pretix/standalone Docker image on Ubuntu 22.04

### Version

2024.6.0

**Comment by raphaelm** [comment]
Sorry, this somehow got lost in my github notifications. Do you still have this issue? How sure are you that you set up the cronjobs for pretix correctly?

**Comment by ginkel** [comment]
Oops... I guess the cronjobs weren't enabled so far. Is there a way to make sure that Pretix won't send hundreds of outdated emails for events that happened long ago once I set up the cronjob?

**Reply by raphaelm** [reply]
Not officially… You could try setting sent_to_organizer to true on all rows of pretixbase_invoice. 

We're bringing in #4550 to make sure this won't happen more, as we've seen this a few times recently.
